site/docs/master/httpproxy.md

```diff
@@ -252,6 +252,54 @@ The TLS **Minimum Protocol Version** a vhost should negotiate can be specified b
 - 1.2
 - 1.1 (Default)
 
+##### Fallback Certificate
+
+Contour provides virtual host based routing, so that any TLS request is routed to the appropriate service based on the hostname specified in the request's HOST header.
+As HOST Header is encrypted during TLS handshake, it can’t be used for virtual host based routing unless client sends HTTPS request specifying hostname using the SNI or the request is first decrypted using a default TLS certificate.
+Some users need to support clients which may not be using SNI.
```

Comment:
I think these 2 sentence could be clarified. The issue here is that legacy TLS client don't send the server name, so we don't know how to select the right certificate, therefore a fallback certificate is needed for these clients (i.e. what you say in the last sentence)